full-stack-tests/core/src/frontend/TestUtility.ts

```diff
@@ -63,15 +65,16 @@ export class TestUtility {
 
     let authorizationClient: AuthorizationClient | undefined;
     if (NativeApp.isValid) {
-      authorizationClient = new NativeAppAuthorization({ clientId: "testapp", redirectUri: "", scope: "" });
+      authorizationClient = new ElectronAppAuthorization ();
       IModelApp.authorizationClient = authorizationClient;
-      const accessToken = await getAccessTokenFromBackend(user);
+      const accessToken = await setBackendAccessToken(user, {
+        clientId: process.env.IMJS_OIDC_ELECTRON_TEST_CLIENT_ID ?? "",
+        redirectUri: process.env.IMJS_OIDC_ELECTRON_TEST_REDIRECT_URI ?? "",
+        scope: process.env.IMJS_OIDC_ELECTRON_TEST_SCOPES ?? "",
+      });
       if ("" === accessToken)
         throw new Error("no access token");
```

Comment:
I think you'll need to call the new method from certaCommon here.

Reply:
Do you within the if-block?

Reply:
Nope, I'd do it below there. It just didn't let me leave a comment below this line :)